.github/workflows/release.yaml

```diff
@@ -16,8 +16,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: echo upload_url
```

Comment:
These commands didn't do anything; the upload_url seems to be empty 🤷🏼‍♂️

Reply:
Don't hesitate to troubleshoot on the public slack; maybe Stew will have ideas

Reply:
I'm pretty sure they're just vestigial 😄 ;

But if @stew  or @ceedubs  know otherwise, feel free to let me know 😄

Reply:
Yeah I'm pretty sure these were serving no purpose 👍